tools/eslint-rules/crypto-check.js

```diff
@@ -70,7 +75,16 @@ module.exports = function(context) {
 
   function report(nodes) {
     nodes.forEach((node) => {
-      context.report(node, msg);
+      context.report({
+        node,
+        message: msg,
+        fix: (fixer) => {
+          return fixer.insertTextAfter(
+            commonModuleNode,
```

Comment:
Need help here. ```commonModuleNode``` comes as null here when running ```make -j4 test```. I'm setting ```commonModuleNode ``` in ```testCryptoUsage``` method.

PS: This works fine when running eslint with ```--fix``` to fix a generated error.

Reply:
Ping @apapirovski